Vagrantfile

```diff
@@ -56,8 +56,8 @@ Vagrant.configure(2) do |config|
     config.vm.box = "elastic/oraclelinux-7-x86_64"
     rpm_common config
   end
-  config.vm.define "fedora-24" do |config|
-    config.vm.box = "elastic/fedora-24-x86_64"
+  config.vm.define "fedora-25" do |config|
```

Comment:
My two arguments against this are:

1. More vms increase (measurably) the time that the packagingTests jobs take to complete.
2. Proposed [GA for FC-26](https://fedoraproject.org/wiki/Releases/26/Schedule) is, as of now, 2017-06-27. This means that one month after i.e. somewhere in July FC-24 will be EOL, which is not too far away.

Other than the two arguments above, I am happy to keep it.